pbkit_ext.h

```diff
@@ -14,6 +14,11 @@
 // "Beta4" class for blending operations (see xf86-video-nouveau).
 #define GR_CLASS_72 0x72
 
+// Enumerated DMA channel destinations
+#define DMA_CHANNEL_FOR_PIXEL_RENDERER_9 9
```

Comment:
Let's drop the numbers from these constants, they're not actually relevant for users. I also think we can be slightly more terse and drop the `FOR` part.

e.g., `DMA_CHANNEL_PIXEL_RENDERER`

Reply:
I was worried if I didn't include the 9 if you were searching for a DMA by one of the known channel values you'd miss the hit.  But I can whack them if you'd like.  I also wasn't sure if this belonged in pbkit_ext or the nxdk_ext.

Reply:
pbkit_ext is the right home since this is tied directly to the pbkit implementation (I think it could decide to use any DMA channel for this)

In practice searching for the 9 is likely to have so many results that it'd be noise anyway, and if pbkit gets refactored in the future such that DMA_CHANNEL_PIXEL_RENDERER becomes `15` you'd end up needing to rename things scattered across the codebase that don't actually care what the value is, just what the logical use case is

Reply:
done